package.json

```diff
@@ -53,8 +53,8 @@
     "depcheck": "lerna exec 'npx depcheck --skip-missing --ignores nan,agent-base,@types/*' --stream"
   },
   "devDependencies": {
-    "@apollo/federation": "^0.38.1",
```

Comment:
Issue: Please add a proper deprecation log to your codebase.

Reply:
Added